apps/masterbots.ai/components/routes/chat/chat-message.tsx

```
@@ -1,35 +1,59 @@
   const cleanMessage = { ...message, content: cleanPrompt(message.content) }
+  const [references, setReferences] = useState<WebSearchResult[]>([])
+
+  const ReferencesSection = () => {
+    if (references.length === 0) return null
+
+    return (
+      <div className="pt-4 mt-4 border-t border-gray-200">
+        <h3 className="mb-2 text-lg font-semibold">References</h3>
+        <div className="space-y-4">
+          {references.map((ref, index) => (
+            <div key={index} className="flex gap-4">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)